### DISCUSSION TITLE
Tutorial step is incorrect

### DISCUSSION THREAD

**Discussion by mslourens** [Budibase/budibase#1290] [Help] [answered]
I am currently trying to learn how to use Budibase, so I am following the tutorial. At [chapter 4](https://docs.budibase.com/tutorial/tutorial-adding-data) there is a step saying `Call the column Logos, and select text as the column type`, but in the following step you have to `add the image by dragging it onto the attachment field or clicking the upload button`. This is not available for column type 'text'. The logos column should be of type 'attachment'. 

I'm not sure if this is the right place to mention this, but I didn't think it was an actual bug.

**Comment by markdwoods3** [comment, marked as answer]
You are right, it looks like a typo so that should be amended.

If you watch the video the demonstrator selects Type 'Attachment' which confirms the above.

**Comment by joebudi** [comment]
You are correct. This is indeed an issue. I have resolved this by changing Text to Attachments. Thank you for finding this issue and raising it here. I will add, the tutorial is dated. It should still provide direction and a good overview of the platform. We will update the tutorial once we have implemented our next phase of work - which will bring pretty big changes to the platform.
